components/PostCard.tsx

```diff
@@ -16,7 +17,15 @@ const PostCard = ({ post }: Props) => {
         <div className="content">
           <Link href={`/post/${post.slug}`} as={undefined}>
             <a className="post-img">
-              <img src={post.featuredImage} alt="blog post image" />
+              <Image
+                src={post.featuredImage}
+                alt="blog post image"
+                layout="fill"
+                objectFit="cover"
+                quality={100}
+                // width={800}
+                // height={400}
```

Comment:
@ntorbinskiy Why you are leaving those comments, they are here for historical reasons, or it should actually be some values?

Reply:
No, I should delete them

Reply:
@ntorbinskiy um okay, but for proper layout rendering without layout shifting, does it need any values?

Reply:
![image](https://user-images.githubusercontent.com/60061661/195800752-40f67457-efe9-4387-a3bb-26d8a39b1b07.png)

Reply:
@ntorbinskiy Um okay, sounds cool, just don't forget to tag when sending a message on GitHub, as I do not receive email messages, and then it gets harder to find those messages

Reply:
@alexander-vashchuk 
ok👍